github-pull-requests.go

```diff
@@ -92,9 +91,7 @@ func populatePRs(rName string, url string) {
 		for _, assignee := range elem.Assignees {
 			eachPRIssue.Assignees += assignee.Login + " "
 		}
-		eachPRIssue.UpdatedAt = elem.UpdatedAt.Format(buzzTimeLayout)
-		delta := elem.UpdatedAt.Sub(elem.CreatedAt)
-		eachPRIssue.Hours = int64(delta.Hours())
+		eachPRIssue.UpdatedAt = elem.UpdatedAt.Unix()
```

Comment:
I think we want to compute : time elapsed since last updated i.e age = now - last_updated.